.github/workflows/CI.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           - stable
           - beta
           - nightly
-          - "1.65.0"
+          - "1.70.0"
```

Comment:
> We lose the CI testing on the minimum supported version, rust 1.65. Are we sure we are ok with that?

No, this needs to be optional and the CI needs to be updated accordingly.

Reply:
>  CI needs to be updated accordingly.

What do toy mean by this?

Reply:
I removed in in the meantime from ci.

Reply:
You need to disable this specific feature for the 1.65 CI run. It would be good to enable it for the others though.

Reply:
Done.